packages/pigeon/run_tests.sh

```diff
@@ -8,6 +8,10 @@
 # exit when any command fails
 set -ex
 
+JAVA_LINTER=checkstyle-8.41-all.jar
```

Comment:
maybe just extract the version rather than the file name? The version is still hardcoded in the url path below